SecondSight/AprilTags/Detector.py

```diff
@@ -162,6 +162,33 @@ def getPosition(img, camera_matrix, dist_coefficients, valid_tags=range(1, 9), c
     return detections
 
 
+# Source: https://math.stackexchange.com/a/1033561
+def fuseApriltags(dets: List[Detection], year: str = '2023') -> tuple[float, float, float]:
```

Comment:
It would be much more elegant to set the year in the constructor.

Reply:
Which constructor?

Reply:
Oh my, this isn't part of the manager class, I wrongly assumed it was. It might make more sense to try to move all functionality into a class to make things more readable